### DISCUSSION TITLE
unknown field "priorityClassName"

### DISCUSSION THREAD

**Discussion by jishaashokan** [actions/actions-runner-controller#1736] [Questions] [answered]
Hello,

Could you please give the correct field for the above? 

This is the error I get : 

unknown field "priorityClassName" in dev.summerwind.actions.v1alpha1.RunnerDeployment.spec.template.spec

**Comment by toast-gear** [comment, marked as answer]
use https://github.com/rewanthtammana/kubectl-fields to find the path
